azure-pipelines.yml

```diff
@@ -134,11 +158,32 @@ jobs:
       cp ../*.deb .
     displayName: "Compile sonic sairedis"
   - script: |
-      make check || true
-    displayName: "Run sonic sairedis unit tests"
+      sudo cp azsyslog.conf /etc/rsyslog.conf
+      sudo service rsyslog restart
+    displayName: "Update rsyslog.conf"
   - publish: $(System.DefaultWorkingDirectory)/
     artifact: sonic-sairedis.arm64
     displayName: "Archive sonic sairedis debian packages"
+  - script: |
```

Comment:
do you still needs this if not running the test?

Reply:
this script is exposing syslog as artefact, so if it could be helpful if build on azure pipeline will fail, since build is always performed, just tests for arm64 and armhf was removed, tests from amd64 still are active